crates/nu-std/src/lib.rs

```diff
@@ -67,9 +67,9 @@ use std pwd
 
         let _ = working_set.add_virtual_path(std_dir, VirtualPath::Dir(std_virt_paths));
 
-        // Change the currently parsed directory
-        let prev_currently_parsed_cwd = working_set.currently_parsed_cwd.clone();
-        working_set.currently_parsed_cwd = Some(PathBuf::from(NU_STDLIB_VIRTUAL_DIR));
+        // Add a placeholder file to the stack of scripts being processed.
+        let placeholder = [NU_STDLIB_VIRTUAL_DIR, "placeholder"].iter().collect();
```

Comment:
Instead of “placeholder” I'd choose some different name, like “loading_stdlib”. The identical file could also be parsed to the `parse()` below.